src/durableorchestrationclient.ts

```diff
@@ -648,42 +638,36 @@ export class DurableOrchestrationClient {
      * existing instance will be silently replaced by this new instance.
      * @param orchestratorFunctionName The name of the orchestrator function to
      *  start.
-     * @param instanceId The ID to use for the new orchestration instance. If
-     *  no instanceId is specified, the Durable Functions extension will
-     *  generate a random GUID (recommended).
-     * @param input JSON-serializable input value for the orchestrator
-     *  function.
+     * @param options optional object specifying the orchestration instance ID and any input to the orchestration

```

Comment:
This parameter may grow over time. So I suggest this more general wording:

```suggestion
     * @param options optional object to control the scheduled orchestrator (e.g provide input, instanceID)

```

Reply:
Done